packages/thumbprint-react/components/ServiceCard/index.tsx

```diff
@@ -1,10 +1,35 @@
 import React, { forwardRef } from 'react';
-import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import Image from '../Image/index';
 import styles from './index.module.scss';
 
-const ServiceCardImage = forwardRef((props, outerRef) => {
+type ImageSource = {
+    type: 'image/webp' | 'image/jpeg' | 'image/png' | 'image/gif';
+    srcSet: string;
+};
+
+interface ServiceCardImagePropTypes {
+    /**
+     * URL pointing to image to be displayed. This image must have an aspect ratio of 8:5.
+     * @deprecated Use `src` instead of `url`.
+     */
+    url?: string;
+    /**
+     * URL pointing to image to be displayed. This image must have an aspect ratio of 8:5.
+     */
+    src: string;
+    /**
+     * Allows the browser to choose the best file format and image size based on the device screen
+     * density and the width of the rendered image. Images must have an aspect ratio of 8:5.
+     */
+    sources?: ImageSource[];
+    /**
+     * Image alt tag that's passed to `aria-label` for better accessibility.
+     */
+    alt?: string;
+}
+
+const ServiceCardImage = forwardRef<HTMLElement, ServiceCardImagePropTypes>((props, outerRef) => {
```

Comment:
Should the anonymous function be typed?

Reply:
React generates them automatically using the `forwardRef` generic arguments `<HTMLElement, ServiceCardImagePropTypes>`. the types for `props` comes from `ServiceCardImagePropTypes` and the types for outerRef comes from `HTMLElement`